OtripleS.Web.Api/Migrations/20200704230926_OtripleSDb.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace OtripleS.Web.Api.Migrations
+{
+    public partial class OtripleSDb : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
+        }
```

Comment:
Why do we need those files V?

Reply:
removing them

Reply:
Added with migration. But removed them now. Please let me know if any more changes needed in this PR